paths/organization_job_template_locales/create.yaml

```diff
@@ -0,0 +1,93 @@
+---
+summary: Create an organization job template locale
+description: Create a new organization job template locale.
+operationId: organization_job_template_locales/create
+tags:
+- Organization Job Template Locales
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+- "$ref": "../../parameters.yaml#/account_id"
+- "$ref": "../../parameters.yaml#/job_template_id"
+responses:
+  '201':
+    description: Created
+    content:
+      application/json:
+        schema:
+          "$ref": "../../schemas/job_template_locale.yaml#/job_template_locale"
+    headers:
+      X-Rate-Limit-Limit:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Limit"
+      X-Rate-Limit-Remaining:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Remaining"
+      X-Rate-Limit-Reset:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Reset"
+  '400':
+    "$ref": "../../responses.yaml#/400"
+  '404':
+    "$ref": "../../responses.yaml#/404"
+  '429':
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/job_templates/:job_template_id/locales" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X POST \
+      -d '{"locale_name":"de-1","locale_code:"de-DE",user_ids":["abcd1234cdef1234abcd1234cdef1234"],"reviewer_ids":["abcd1234cdef1234abcd1234cdef1234"],"translator_team_ids":["abcd1234cdef1234abcd1234cdef1234"]}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase organization_job_template_locales create \
+    --account_id <account_id> \
+    --job_template_id <job_template_id> \
+    --data '{"locale_name": "de-1", "locale_code: "de-DE", "user_ids": "abcd1234cdef1234abcd1234cdef1234", "reviewer_ids": "abcd1234cdef1234abcd1234cdef1234", "translator_team_ids": "abcd1234cdef1234abcd1234cdef1234"}' \
+    --access_token <token>
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: organization_job_template_locales/create/parameters
+        properties:
+          locale_name:
+            description: locale name
+            type: string
+            example: de-1
+          locale_code:
+            description: locale code
+            type: string
+            example: de-DE
+          user_ids:
+            description: Array of user ids to be assigned to the job template locale
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          reviewer_ids:
+            description: Array of reviewer ids to be assigned to the job template locale
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          translator_team_ids_ids:
```

Comment:
This should be just `translator_team_ids` (extra _ids)

Reply:
"Find & replace" fail 🤦 Thanks!